lib/mayaUsd/render/vp2RenderDelegate/draw_item.h

```diff
@@ -64,8 +64,13 @@ class HdVP2DrawItem final : public HdDrawItem
         //! Whether or not the render item is enabled
         bool _enabled { true };
 
+        //! Whether or not the render item is transparent
+        bool _transparent { false };
+
         //! Primitive type of the render item
-        MHWRender::MGeometry::Primitive _primitiveType { MHWRender::MGeometry::kInvalidPrimitive };
+        MHWRender::MGeometry::Primitive _primitiveType {
+            MHWRender::MGeometry::kInvalidPrimitive
+        }; // TODO: this is stored on the MRenderItem, do we need it here too?
```

Comment:
Yea, maybe we can remove the duplicate data here because we have the render item pointer already which we can use to query the state and test whether it needs to be updated.